contracts/leaser/src/migrate.rs

```diff
@@ -2,12 +2,11 @@ use lease::api::MigrateMsg;
 use platform::batch::Batch;
 use sdk::cosmwasm_std::Addr;
 
-use crate::{msg::MaxLeases, result::ContractResult};
-
-pub struct Customer<LeaseIter> {
-    customer: Addr,
-    leases: LeaseIter,
-}
```

Comment:
1. What is the motivation to include this change in this PR?
2. The previous name was descriptive enough. What if tomorrow we add another attribute to the customer?

Reply:
1) It needed to be moved because it was used by code outside the migration one, which is guarded by a feature. This was part of making sure everything compiles just fine on it's own.
2) I suppose it was. Perhaps I could revert it.

Reply:
1. if I am not mistaken we do not change the features around this code
2. please do